hyp3_autorift/io.py

```diff
@@ -60,24 +44,34 @@ def _get_s3_keys_for_dem(prefix=AUTORIFT_PREFIX, dem='GRE240m'):
         'yMinChipSize',
         'xMaxChipSize',
         'yMaxChipSize',
-        # FIXME: Was renamed from masks to sp by JPL; change hasn't been propagated to autoRIFT
-        #        keep last so we can easily rename the file after downloading
         'sp',
     ]
     keys = [f'{prefix}/{dem}_{tag}.tif' for tag in tags]
     return keys
 
 
-def fetch_jpl_tifs(dem='GRE240m', target_dir='DEM', bucket=ITS_LIVE_BUCKET, prefix=AUTORIFT_PREFIX):
-    # FIXME: gdalwarp  needed subset instead?
-    log.info(f"Downloading {dem} tifs from JPL's AWS bucket")
+def subset_jpl_tifs(polygon: ogr.Geometry, buffer: float = 0.15, dem: str = 'NPS_0240m', target_dir: str = 'DEM',
```

Comment:
different default dem than `_get_s3_keys_for_dem()`, consider making both the same?  or both having no default?

Reply:
I took out the defaults -- doesn't really make sense to have a default DEM anymore.